rfcs/css_build_system.md

```diff
@@ -0,0 +1,15 @@
+# RFC 136: Remove the CSS build system
+
+## Summary
+
+Remove css/tools/ and the special lint rules for css/.
+
+## Details
+
+http://test.csswg.org/suites/ hasn't been updated since January.
+
+TODO: does the CSSWG want to keep the spec link rule?
```

Comment:
I would proceed for now on the assumption the answer is "yes", but it's worth asking the question. If not we can always remove the requirement in a second step.

Reply:
I've done it in a separate commit. Let's just say it's not part of this RFC to reduce risk.